blog/electron-27-0.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Electron 27.0.0
+date: 2023-10-10T00:00:00.000Z
+authors:
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-27-0
+---
+
+Electron 27.0.0 has been released! It includes upgrades to Chromium `118.0.5993.32`, V8 `11.8`, and Node.js `18.17.1`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 27.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on Twitter, or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Stack Changes
+
+- Chromium `118.0.5993.32`
+  - [New in Chrome 118](https://developer.chrome.com/blog/new-in-chrome-118/)
+  - [New in DevTools 118](https://developer.chrome.com/blog/new-in-devtools-118/)
+- Node.js `18.17.1`
+  - [Node 18.17.1 blog post](https://nodejs.org/en/blog/release/v18.17.1/)
+- V8 `11.8`
+
+### Breaking Changes
+
+### Removed: Extended support for Windows 7/8/8.1
+
+Electron 22 is now end of life (see [Farewell, Windows 7/8/8.1](https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice) for more details). All currently supported versions of Electron have dropped support for Windows 7/8/8.1.
```

Comment:
I'm not sure if this should be in a different section, this skim reads as "Electron 27 dropped support", moving it into the "end of extended support" section at the bottom might be appropriate

Reply:
Yeah, I was waffling on whether or not we should mention it here - removed here and left in the end of extended support section at the end 🙂